app/src/main/java/com/iitp/njack/iitp_connect/core/youtube/VideoActivity.java

```diff
@@ -0,0 +1,216 @@
+package com.iitp.njack.iitp_connect.core.youtube;
+
+import android.Manifest;
+import android.accounts.AccountManager;
+import android.app.Dialog;
+import android.arch.lifecycle.Observer;
+import android.arch.lifecycle.ViewModelProviders;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.databinding.DataBindingUtil;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.Toast;
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.GoogleApiAvailability;
+import com.google.api.client.googleapis.extensions.android.gms.auth.GoogleAccountCredential;
+import com.google.api.client.util.ExponentialBackOff;
+import com.google.api.services.youtube.YouTubeScopes;
+import com.iitp.njack.iitp_connect.R;
+import com.iitp.njack.iitp_connect.databinding.ActivityVideoBinding;
+import java.util.Arrays;
+import java.util.List;
+import pub.devrel.easypermissions.AfterPermissionGranted;
+import pub.devrel.easypermissions.EasyPermissions;
+
+public class VideoActivity extends AppCompatActivity {
+    static final int REQUEST_ACCOUNT_PICKER = 1000;
+    static final int REQUEST_AUTHORIZATION = 1001;
+    static final int REQUEST_GOOGLE_PLAY_SERVICES = 1002;
+    static final int REQUEST_PERMISSION_GET_ACCOUNTS = 1003;
+    private static final String PREF_ACCOUNT_NAME = "accountName";
+    private static final String[] SCOPES = {YouTubeScopes.YOUTUBE_READONLY};
+    GoogleAccountCredential googleAccountCredential = null;
+    String playlist_id = "";
+    private VideoViewModel videoViewModel;
```

Comment:
Have you even cared enough to read the code that is already there in the application. I highly doubt it. See how we have used DI for inserting view models.